e2e_test/source_inline/kafka/avro/name_strategy.slt

```diff
@@ -36,16 +36,16 @@ WITH (
 FORMAT UPSERT ENCODE AVRO (schema.registry = '${RISEDEV_SCHEMA_REGISTRY_URL}');
 
 
-# TODO: Uncomment this when we add test data kafka key with format `"ID":id`
-# statement ok
-# CREATE TABLE upsert_avro_json (
-#   PRIMARY KEY("ID")
-# )
-# WITH (
-# connector = 'kafka',
-#  properties.bootstrap.server = '${RISEDEV_KAFKA_BOOTSTRAP_SERVERS}',
-# topic = 'upsert_avro_json')
-# FORMAT UPSERT ENCODE AVRO (schema.registry = '${RISEDEV_SCHEMA_REGISTRY_URL}');
+statement ok
+CREATE TABLE upsert_avro_json (
+  PRIMARY KEY("ID_KEY")
+)
+INCLUDE KEY AS "ID_KEY"
```

Comment:
Isn't this the same as above now? I believe the original intention is trying to use a column as pk instead of kafka key as pk

Reply:
Yes, it is. In the very beginning, we can infer schema and pk from the schema registry: 1) we compare the key schema and value schema, 2) make sure the key schema is a subset of the value ones, and 3) take the key schema as the primary key.
But later we deprecate the behavior and do not interpret the key part, just taking it as bytes, pivoting to additional column solution. 

* rename: column "ID" is also contained in the value part so we have to rename the additional column to another name.

Reply:
> Yes, it is.

If it's the same, why not delete this part???

Reply:
hummm make sense, I thought the case was for `except` syntax

Reply:
I finally got it. I thought the original test is using one value column as key, but you mean it's one column in key schema.